server/src/actors/events.rs

```diff
@@ -38,9 +39,15 @@ macro_rules! clone {
 }
 
 #[derive(Debug, Fail)]
-pub enum ProcessingError {
+pub enum EventError {
```

Comment:
With this change I tried to reduce the API surface of the `EventManager` actor. Most of the error kinds are only thrown internally during processing (and used for logging), so there's no need to expose them.